.github/workflows/auto-release.yaml

```diff
@@ -0,0 +1,31 @@
+name: Auto Release
+
+on:
+  push:
+    branches:
+      - main
+
+
+jobs:
+  auto-release:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Download auto
+        run: |
+          wget -O- https://github.com/intuit/auto/releases/download/v10.37.1/auto-linux.gz | gunzip > ~/auto
```

Comment:
will it be very slow to download from github release, use our oss bucket?

Reply:
In the test repo, the whole process will ends within one minute.  The GitHub Action runner have no GFW problem. :D